mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -107,17 +126,21 @@ CLASS none/cn net/minecraft/util/math/BlockPos
 	METHOD b down ()Lnone/cn;
 	METHOD b up (I)Lnone/cn;
 		ARG 0 distance
-	METHOD b (Lnone/cn;Lnone/cn;)Ljava/lang/Iterable;
-		ARG 0 min
-		ARG 1 max
+	METHOD b iterateBoxPositionsMutable (Lnone/cn;Lnone/cn;)Ljava/lang/Iterable;
+		ARG 0 pos1
+		ARG 1 pos2
 	METHOD b subtract (Lnone/dk;)Lnone/cn;
 		ARG 0 vec
 	METHOD c north ()Lnone/cn;
 	METHOD c down (I)Lnone/cn;
 		ARG 0 distance
+	METHOD c crossProduct (Lnone/dk;)Lnone/cn;
+		ARG 0 vec
 	METHOD d south ()Lnone/cn;
 	METHOD d north (I)Lnone/cn;
 		ARG 0 distance
+	METHOD d crossProduct (Lnone/dk;)Lnone/dk;
```

Comment:
...that's odd. I can check this in the morning but at least one of those does a cross product.